docs/ruleset_functions/is_hvac_sys_fluid_loop_attached_to_boiler(hvac_b.id).md

```diff
@@ -0,0 +1,25 @@
+# is_hvac_sys_fluid_loop_attached_to_boiler  
+
+**Description:** Returns TRUE if the fluid loop associated with the heating system associated with the HVAC system is attached to a boiler. Returns FALSE if this is not the case.   
+
+**Inputs:**  
+- **B-RMR**: To evaluate if the fluid loop associated with the heating system associated with the HVAC system is attached to a boiler.   
+- **hvac_b.id**: The id of the hvac system to evaluate.  
+
+**Returns:**  
+- **is_hvac_sys_fluid_loop_attached_to_boiler**: Returns TRUE if the fluid loop associated with the heating system associated with the HVAC system is attached to a boiler. Returns FALSE if this is not the case.   
+ 
+**Function Call:** None  
+
+## Logic:   
+- Set is_hvac_sys_fluid_loop_attached_to_boiler equal to FALSE: `is_hvac_sys_fluid_loop_attached_to_boiler = FALSE`  
+- For each boiler in B_RMR, save boiler to loop boiler dictionary: `for boiler_b in B_RMR.RulesetModelInstance.boilers: loop_boiler_dict[boiler_b.loop].append(boiler_b)`
+- Create an object associate with the heating_system fluid loop associated with hvac_b: `fluid_loop_b = hvac_b.heating_system[0].hot_water_loop`
```

Comment:
May need to be changed if we decide HW/CHW coils would be specified at the terminal level for Sys 1.

Reply:
as decided - systems with compressors would use HVAC system and hence the coils would specified at HVAC system level for system 1.